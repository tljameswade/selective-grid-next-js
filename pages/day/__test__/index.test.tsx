import renderer from 'react-test-renderer';
import DayPage from '../index.page';

describe('Day page', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<DayPage />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});