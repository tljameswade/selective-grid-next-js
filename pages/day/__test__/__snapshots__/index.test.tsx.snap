// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Day page renders correctly 1`] = `
<div>
  Hello Day. For testing ssh keys.
</div>
`;

exports[`renders correctly 1`] = `
<div>
  Hello Day. For testing ssh keys.
</div>
`;
